cli.js

```diff
@@ -72,6 +73,9 @@ codemods.sort(utils.sortByVersion);
 
 var versions = utils.getVersions(codemods);
 
+var avaConf = pkgConf.sync('ava');
```

Comment:
Yup, confirming, seems to work fine without a `package.json` file, and any other configuration I mentioned above.